.github/workflows/push.yml

```
@@ -22,8 +22,9 @@
   MYSQL_ROOT_PASSWORD: root
 
 jobs:
-  Test-Application-Code-Coverage:
+  Test-Application:
```

actionlint (1.7.4)

job "test-application" needs job "code-quality-checks" which does not exist in this workflow

(job-needs)